src/Features/CSharp/Portable/ReplacePropertyWithMethods/ConvertValueToReturnsRewriter.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace Microsoft.CodeAnalysis.CSharp.ReplacePropertyWithMethods
+{
+    internal partial class CSharpReplacePropertyWithMethodsService
+    {
+        private class ConvertValueToReturnsRewriter : CSharpSyntaxRewriter
+        {
+            public static readonly CSharpSyntaxRewriter Instance = new ConvertValueToReturnsRewriter();
```

Comment:
Fields should start with a lowercase letter, right? At a minimum, the capitalization of this is inconsistent between C# and VB.